tests/tests_basic_ipa.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: Install IPA server
+  hosts: all
+  become: true
+  tasks:
+    - import_tasks: tasks/setup_ipa.yml
+
+- name: Issue simple self-signed certificate
+  hosts: all
+  become: true
+
+  vars:
+    certificate_requests:
+      - name: mycert
+        dns: ipaserver.test.local
+        principal: HTTP/ipaserver.test.local@TEST.LOCAL
+        ca: ipa
+  roles:
+    - linux-system-roles.certificate
+
+- name: Verify certificate
+  hosts: all
+  become: true
+  vars:
+    certificates:
+      - path: /etc/pki/tls/certs/mycert.crt
+        key_path: /etc/pki/tls/private/mycert.key
+        subject:
+          - O=TEST.LOCAL
+          - CN=ipaserver.test.local
+        subject_alt_name:
+          - DNS:ipaserver.test.local
+        # IPA CA doesn't respect the requested key_usage (and that's ok)
```

Comment:
Is it something to be noted in README https://github.com/linux-system-roles/certificate/pull/30? (unless it's a common knowledge...)

Reply:
Actually it is in the README. It's under section: https://github.com/linux-system-roles/certificate#certificate_requests
It's a generic note because it's pretty hard to know what are the exact fields a CA will or will not honor. I've added this extra comment in the tests so it doesn't look like we are performing a wrong verification. 
Do you think I should be more specific and maybe mention the README there?

Reply:
Thank you, @seocam! I think this is clear enough.